nixos/modules/services/mail/spamassassin.nix

```diff
@@ -126,6 +126,8 @@ in
     };
 
     systemd.services.sa-update = {
+      wants = [ "network-online.target "];
+      after = [ "network-online.target "];
```

Comment:
```suggestion
      wants = [ "network-online.target" ];
      after = [ "network-online.target" ];
```

Reply:
Sorry, I have no idea how this happened. Fixed.